sklearn/svm/liblinear.pxd

```diff
@@ -20,10 +20,10 @@ cdef extern from "src/liblinear/linear.h":
 cdef extern from "src/liblinear/liblinear_helper.c":
     void copy_w(void *, model *, int)
     parameter *set_parameter(int, double, double, int, char *, char *, int, int, double)
-    problem *set_problem (char *, char *, np.npy_intp *, double)
+    problem *set_problem (char *, char *, np.npy_intp *, double, char *)
```

Comment:
IIRC, it abstracts away the difference between dense and sparse Liblinear. That difference doesn't exist here.